csp-mixin/dashboards.libsonnet

```diff
@@ -27,6 +27,28 @@ local commonlib = import 'common-lib/common/main.libsonnet';
         ),
     }
     +
+    {
+      [csplib.config.uid + '-loadbalancer.json']:
+        local variables = csplib.signals.loadbalancer.getVariablesMultiChoice();
+        g.dashboard.new(csplib.config.dashboardNamePrefix + 'Load Balancing')
+        + g.dashboard.withUid(csplib.config.uid + '-loadbalancer')
+        + g.dashboard.withTags(csplib.config.dashboardTags)
+        + g.dashboard.withTimezone(csplib.config.dashboardTimezone)
+        + g.dashboard.withRefresh(csplib.config.dashboardRefresh)
+        + g.dashboard.timepicker.withTimeOptions(csplib.config.dashboardPeriod)
+        + g.dashboard.withVariables([
+          if std.asciiLower(v.label) == std.asciiLower(csplib.config.blobStorage.bucketLabel)  //v.label == 'Bucket_name'
+          then v { label: 'Bucket Name' }
+          else v
+          for v in variables
+        ])
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            csplib.grafana.rows.glb_requests
+          )
+        ),
+    }
+    +
     if csplib.config.uid == 'azure' then
```

Comment:
Since the GCP loadbalancer dashboard is GCP only (for now) we'll need to add a conditional like this for dashboards which are GCP only.

The first dashboard, for blob storage, is Azure and GCP. So when you append your new dashboard to that list, it expects that there will be signals for both sources.

I can add a commit which makes this modification.